docs/check.Dockerfile

```diff
@@ -0,0 +1,43 @@
+
+FROM alpine:3.9 as alpine
```

Comment:
maybe do `ruby:2-alpine3.10` instead and it would save you the extra install below

Reply:
There are some inconsistency when I change the image. Since we don't use it at this moment in travis, we can leave it as it is to have the base ready and then we optimize it later.